### DISCUSSION TITLE
Time to drop Python 3.6?

### DISCUSSION THREAD

**Discussion by cclauss** [xlcnd/isbnlib#118] [General]
Next week Python 3.11 will be released and we are already testing release candidate 2 on GitHub Actions.

Supporting six different versions of CPython is a big ask.  Is it time to drop support for Python 3.6?
* https://devguide.python.org/versions

**Comment by xlcnd** [comment]
I am working on a new version of `isbnlib` (series 4.X.X) that will break compatibility and is an opportunity to modernize the code base and reconsider the versions of python supported. 

However, I would like to keep the series 3.X.X as compatible as possible with older versions.
